src/main.rs

```diff
@@ -53,9 +52,14 @@ fn main() -> Result<(), io::Error> {
         if opt.restore {
             let info_path = trashinfo::info_path(&file_absolute.unwrap());
             let trash_info = trashinfo::Trashinfo::from_file(&info_path)?;
-            let destination = trash_info.1.clone();
+            let mut destination = trash_info.1.clone();
+            let dest_path = Path::new(&destination);
+            if dest_path.exists() {
+                let noclobber_suffix = &chrono::NaiveDateTime::parse_from_str(&trash_info.2, "%Y-%m-%dT%H:%M:%S").expect("Could not parse datetime from trashinfo.").format("_%H%M%S-%y%m%d").to_string();
```

Comment:
@Jammyjamjamman This variable already exists (see L27)

Reply:
I think this shouldn't cause a problem, due to Rust's [shadowing rules](https://doc.rust-lang.org/stable/rust-by-example/variable_bindings/scope.html). But if it's too ugly I'd be happy to change it.

Reply:
But why wouldn't you just use the variable that's already been created on L27, instead of recreating one with the same name?

Reply:
The other noclobber suffix gives `now`'s datetime. I was trying to give it the same date as in its `.trashinfo`. Or do we want to use the current datetime as the suffix?

Reply:
They should both be `now` but different format.

The trashinfo spec requires "%Y-%m-%dT%H:%M:%S", which is written to the .trashinfo file.

The no_clobber suffix format implemented on L27 ("_%H%M%S-%y%m%d") is one I've been using for rmw for several years. It just assures that destination files don't clobber existing files with the same name when either being restored or rmw'ed. 

It's not in the .trashinfo spec. no_clobber uses a slightly shorter format, which is sufficient. unless someone manages to run rmw (or rmwrs) more than once within the same second. It doesn't need to match the same format as the DeletionDate.

It's explained a bit more in the [rmw README](https://github.com/theimpossibleastronaut/rmw/blob/master/README.md)